pkg/crc/preflight/preflight_windows.go

```diff
@@ -7,13 +7,15 @@ var hypervPreflightChecks = [...]PreflightCheck{
 		configKeySuffix:  "check-administrator-user",
 		checkDescription: "Checking if running as normal user",
 		check:            checkIfRunningAsNormalUser,
-		fix:              fixRunAsNormalUser,
+		fixDescription:   "crc should be ran as a normal user",
+		flags:            NoFix,
 	},
 	{
 		configKeySuffix:  "check-windows-version",
-		checkDescription: "Checking Windows 10 release",
+		checkDescription: "Checking  Windows 10 release",
```

Comment:
Minor: extra space `Checking Windows`

Reply:
Fixed, thanks, I thought I had fixed it already, but I actually reintroduced it while rebasing.